lib/enumerations/finder_methods.rb

```diff
@@ -31,7 +31,7 @@ def where(**args)
     #   Role.find_by(name: 'Admin') => #<Enumerations::Value: @base=Role, @symbol=:admin...>
     #
     def find_by(**args)
-      where(args).first
+      where(**args).first
```

Comment:
https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

`#where` is defined as `def where(**args)` so we need to splat arguments first.